tests/settings.test.ts

```diff
@@ -1,25 +1,24 @@
 import { expect, test } from '@playwright/test';
-import { mockTagsResponse } from './mocks';
+import { MOCK_API_TAGS_RESPONSE, mockTagsResponse } from './utils';
 
 test.beforeEach(async ({ page }) => {
 	await mockTagsResponse(page);
 });
 
 test('displays model list and updates settings store', async ({ page }) => {
+	const modelSelect = page.getByLabel('Model');
+	
 	await page.goto('/');
 
 	// Check if the model list contains the expected models
-	const modelNames = await page.$$eval('select#model option', options =>
-		options.map(option => option.textContent?.trim())
-	);
-	expect(modelNames).toContain('gemma:7b');
-	expect(modelNames).toContain('openhermes2.5-mistral:latest');
+	await expect(modelSelect).toContainText(MOCK_API_TAGS_RESPONSE.models[0].name);
+	await expect(modelSelect).toContainText(MOCK_API_TAGS_RESPONSE.models[1].name);
```

Comment:
Referencing model names from the `MOCK_API_TAGS_RESPONSE` so we can update the model names in the future without having to find/replace across all of the tests.